src/ripple/ledger/impl/View.cpp

```diff
@@ -126,46 +126,53 @@ accountHolds (ReadView const& view,
         auto const reserve =
             view.fees().accountReserve(
                 sle->getFieldU32(sfOwnerCount));
-        auto const balance =
-            sle->getFieldAmount(sfBalance).xrp ();
+
+        auto const fullBalance =
+            sle->getFieldAmount(sfBalance);
+
+        auto const balance = view.balanceHook(
```

Comment:
The name `balanceHook()` does not really describe what this method does.  I'm guessing that a name like `availableBalance()` would be more descriptive?